### ISSUE TITLE
Apache Drill datasource

### ISSUE THREAD

**Issue by Magmatrix** [grafana/grafana#88216]
**Why is this needed**:

I am using Apache Drill to query large sets of data (100 TB +) from a storage cluster (Parquet currently).
I need a datasource connection to Drill to be able to pull this data into Grafana.
With Drill it's easy to write SQL queries over multiple types of data stores, but there is currently no way to get the result into Grafana.
(atm i have to use Superset to visualize the results, which suck)

**What would you like to be added**:

Add "Apache Drill" as a datasource plugin.

**Who is this feature for?**

Data mining, analytical queries, queries spanning over large datasets and different types of storage.

**Comment by Magmatrix** [comment]
Maybe pick this one up and see if it can be modified to work with the current version of Grafana?
https://github.com/benkindt/grafana-drill-datasource

**Comment by Sam-H101** [comment]
I agree this would be nice to have. I was researching Apache Drill earlier today and it would be a big benefit if Grafana could incorporate this into its platform.

**Comment by williamhatcher** [comment]
Ditto. Though I may take a look at the repo and see about adding it.
